Resources/views/Collector/db.html.twig

```diff
@@ -186,55 +162,83 @@
 
     <h2>Database Connections</h2>
 
-    {% if collector.connections %}
-        {% include 'WebProfilerBundle:Profiler:table.html.twig' with {data: collector.connections} only %}
+    {% if not collector.connections %}
+        <div class="empty">
+            <p>There are no configured database connections.</p>
+        </div>
     {% else %}
-        <p>
-            <em>No connections.</em>
-        </p>
+        {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.connections, labels: ['Name', 'Service'] }, with_context = false ) }}
     {% endif %}
 
     <h2>Entity Managers</h2>
 
-    {% if collector.managers %}
-        {% include 'WebProfilerBundle:Profiler:table.html.twig' with {data: collector.managers} only %}
+    {% if not collector.managers %}
+        <div class="empty">
+            <p>There are no configured entity managers.</p>
+        </div>
     {% else %}
-        <p>
-            <em>No entity managers.</em>
-        </p>
+        {{ include('WebProfilerBundle:Profiler:table.html.twig', { data: collector.managers, labels: ['Name', 'Service'] }, with_context = false ) }}
     {% endif %}
 
     <h2>Second Level Cache</h2>
 
-    {% if collector.cacheCounts %}
-        {% include 'WebProfilerBundle:Profiler:table.html.twig' with {data: collector.cacheCounts} only %}
+    {% if not collector.cacheEnabled %}
+    <div class="empty">
+        <p>Second Level Cache is not enabled.</p>
+    </div>
+    {% else %}
+        {% if collector.cacheCounts %}
```

Comment:
you should add an info saying that profiling was not enabled for the SLC in the else clause (in the current version of the code, this case also shows ``No cache.`` for that case, which was crappy (it was not checking ``collector.cacheEnabled``)